utils/android/build-toolchain

```diff
@@ -1,48 +0,0 @@
-#!/usr/bin/env bash
-#
-# android/build-toolchain
-#
-# This source file is part of the Swift.org open source project
-#
-# Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
-# Licensed under Apache License v2.0 with Runtime Library Exception
-#
-# See https://swift.org/LICENSE.txt for license information
-# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
-
-set -e
-
-SWIFT_PATH="$( cd "$(dirname $0)/../../.." && pwd )" 
-
-ANDROID_NDK_PATH="${ANDROID_NDK_PATH:?Please set the Android NDK path in the ANDROID_NDK_PATH environment variable}"
-ANDROID_ICU_PATH=${SWIFT_PATH}/libiconv-libicu-android
-
-[ -e ${ANDROID_ICU_PATH} ] || exit 1
-
-SWIFT_ANDROID_TOOLCHAIN_PATH="${SWIFT_PATH}/swift-android-toolchain"
-
-rm -rf ${SWIFT_ANDROID_TOOLCHAIN_PATH}
-
-${SWIFT_PATH}/swift/utils/build-script \
-    -R \
-    --android \
-    --android-ndk ${ANDROID_NDK_PATH} \
-    --android-api-level 21 \
-    --android-icu-uc "${ANDROID_ICU_PATH}/armeabi-v7a" \
-    --android-icu-uc-include "${ANDROID_ICU_PATH}/armeabi-v7a/icu/source/common" \
-    --android-icu-i18n "${ANDROID_ICU_PATH}/armeabi-v7a" \
-    --android-icu-i18n-include "${ANDROID_ICU_PATH}/armeabi-v7a/icu/source/i18n" \
```

Comment:
I just went ahead and got rid of this whole file because it hasn't worked in years. The expanded build preset from #38441 should replace this.